sources/osg/LightSource.js

```diff
@@ -18,7 +21,25 @@ LightSource.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit(
     },
     setLight: function ( light ) {
         this._light = light;
+    },
+    setReferenceFrame: function ( value ) {
+        this._referenceFrame = value;
+    },
+    getReferenceFrame: function () {
+        return this._referenceFrame;
+    },
+    computeBound: function ( bsphere ) {
+        Node.prototype.computeBound.call( this, bsphere );
+        if ( this._light !== undefined && this._referenceFrame === TransformEnums.RELATIVE_RF ) {
+            var position = this._light.getPosition();
+            if ( position[ 3 ] !== 0.0 ) {
+                var div = 1.0 / position[ 3 ];
+                bsphere.expandByVec3( Vec3.createAndSet( position[ 0 ] * div, position[ 1 ] * div, position[ 2 ] * div ) );
```

Comment:
ok, makes a bit more sense, So that means we should be able to tranform light postion.
perhaps we miss something that is in osg that does allow that ?
( as unless you transform light position, w will always be 1.0 )